packages/core-services/src/LocalBroker.ts

```diff
@@ -73,6 +73,7 @@ export class LocalBroker implements IBroker {
 
 			this.methods.set(`${namespace}.${method}`, i[method].bind(i));
 		}
+		void instance.started();
```

Comment:
we should call this only after the broker has been started as well

Reply:
That makes sense, I've added a flag to know if the broker has already started. To add more context, the idea here and the problem it has is that the `started()` method within the services is not being called after the broker has already started, so if you start the broker and create a service after that, this was not called.